test/e2e/apps/rc.go

```diff
@@ -38,6 +38,12 @@ import (
 var _ = SIGDescribe("ReplicationController", func() {
 	f := framework.NewDefaultFramework("replication-controller")
 
+	/*
+		Release : v1.9
```

Comment:
I don't see any value in having the release specified here. Nobody will update it and it'll just rot. The rest is ok.

Reply:
I understand, but the reason I proposed this to the conformance group is to track when a conformance test is added or when it is modified. This is a free form string and up to the reviewer to pay attention to it. All this meta data will show up in a document like https://github.com/cncf/k8s-conformance/blob/master/docs/KubeConformance-1.9.md